cloud/digitalocean/reconciler.go

```diff
@@ -213,10 +212,14 @@ func newClusterDefaults(base *cluster.Cluster) *cluster.Cluster {
 	return new
 }
 
-func handleCtrlC(c chan os.Signal) {
-	sig := <-c
-	if sig == syscall.SIGINT {
-		sigCaught = true
-		logger.Critical("Detected SIGINT. Please be patient while kubicorn cleanly exits. Maybe get a cup of tea?")
-	}
+func handleCtrlC(h *signals.Handler) {
```

Comment:
Can we give this a name that corresponds to the POSIX constant we are handling?

`handleSigInt`